Attentive.Unity/Attentive.Unity.sln

```diff
@@ -0,0 +1,19 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.810.20
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Attentive.Unity", "Attentive.Unity\Attentive.Unity.shproj", "{CF60B0BB-04BF-4663-9B5A-672DB0D616F3}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {85569858-280B-4A51-88BB-176DA60039F4}
+	EndGlobalSection
+EndGlobal

```

Comment:
is it possible to use vs code workspace or project instead of full-on visual studio?